contracts/database/orm/constants.go

```diff
@@ -3,6 +3,7 @@ package orm
 const (
 	DriverMysql      Driver = "mysql"
 	DriverPostgresql Driver = "postgresql"
```

Comment:
Could you please add the DEPRECATED tag for this line?

Reply:
Yes